doc/advance_install.rst

```diff
@@ -0,0 +1,267 @@
+
+
+.. note:: Do not forget to put a correct path to the `my.cnf` MySQL credential file in your each config `DATABASES` setting.
+
+Initialiser Pytition ainsi que les bases de données. Vous devez être dans le virtualenv pour entrer les commandes suivantes:
+
+.. code-block:: bash
+  $ export PYTHONPATH="/etc/pytition"
+  $ cd /srv/pytition/www/pytition/pytition
+  $ sudo -u pytition-admin -- DJANGO_SETTINGS_MODULE="admin.config" python3 manage.py migrate
+  $ sudo -u pytition-admin -- DJANGO_SETTINGS_MODULE="admin.config" python3 manage.py collectstatic
+  $ sudo -u pytition-admin -- DJANGO_SETTINGS_MODULE="admin.config" python3 manage.py compilemessages
+  $ sudo -u pytition-admin -- DJANGO_SETTINGS_MODULE="admin.config" python3 manage.py createsuperuser
+  $ sudo -u orga1-user -- DJANGO_SETTINGS_MODULE="orga1.config" python3 manage.py migrate
+  $ sudo -u orga2-user -- DJANGO_SETTINGS_MODULE="orga2.config" python3 manage.py migrate
+
+.. note:: You will be asked to enter a `username`, `email` and `password` for the administrator's
+
+Before trying to configure a web server you can try to see if your configuration is OK by running:
+Par exemple pour orga1:
+
+.. code-block:: bash
+
+  $ DEBUG=1 DJANGO_SETTINGS_MODULE=orga1.config python3 ./manage.py runserver
+
+You can then point your browser to `http://yourdomain.tld:8000` and check that you can see Pytitiont's home page and log-in with your newly created admin account.
+
+.. warning:: If you've set ``USE_MAIL_QUEUE`` to ``True`` and ``MAIL_EXTERNAL_CRON_SET`` to ``False``, running Pytition via ``manage.py runserver`` might not work well since you need to be run via `uwsgi`. Especially emails might not be sent.
+
+.. note:: If you switch ``USE_MAIL_QUEUE`` from ``False`` to ``True`` at some point, you might have to re-run ``python3 manage.py migrate`` to create the database structures needed for the mail queues.
+
+
+Apache + uwsgi + maintenance mode
+=================================
+
+Install uwsgi dependency::
+
+  sudo apt install uwsgi uwsgi-plugin-python3 python3-uwsgidecorators
+
+and enable proxy_uwsgi on apache:
+  sudo a2enmod proxy_uwsgi
+
+Here is an example of Apache configuration that you can put in `/etc/apache2/sites-available/orga1`::
+
+  <VirtualHost *:80>
+
+  ServerName pytition.orga1.org
+  
+  Alias /static /srv/pytition/www/static
+  Proxypass /static !
+  Alias /mediaroot /srv/pytition/www/mediaroot/orga1/
+  Proxypass /mediaroot !
+  
+  ProxyPass / unix:/var/run/uwsgi/app/pytition.orga1.org/socket|uwsgi://localhost/
+
+  <Directory /srv/pytition/www/static>
+  Require all granted
+  </Directory>
+  
+  <Directory /srv/pytition/www/mediaroot>
+  Require all granted
+  </Directory>
+  
+  CustomLog /var/log/apache2/access.log combined
+  CustomLog /var/log/apache2/pytition.orga1.org.log combined
+  
+  </VirtualHost>
+
+Voici un exemple de configuration pour uwsgi à mettre dans /etc/uwsgi/app-available avec un lien dans /etc/uwsgi/app-enabled
```

Comment:
Here is an example of uwsgi configuration that you can put in `/etc/uwsgi/app-available/`. Don't forget to create a symbolic link in `/etc/uwsgi/app-enabled` pointing to the previously created file.